### PR TITLE
have a look around

### PR DESCRIPTION
**Pull request by michaelneale**
Just trying to work out what the jest reports are doing.

### CODE REVIEW

**Comment by michaelneale** [comment]
This is just to help solve: https://issues.jenkins-ci.org/browse/JENKINS-47684

File: Jenkinsfile [hunk 1]
```diff
@@ -37,6 +37,9 @@ node() {
           }
           junit '**/target/surefire-reports/TEST-*.xml'
           junit '**/target/jest-reports/*.xml'
+          sh 'echo JEST STUFF'
+          sh 'ls -lah blueocean-dashboard/target/jest-reports/'
+          archive 'blueocean-dashboard/target/jest-reports/junit.xml'
```
Comment (AlanFoster) [inline Jenkinsfile]: ❔ is `archive` deprecated in favour of `archiveArtifacts` ? 🤔

Comment (i386) [reply Jenkinsfile]: regrettably - `archive` is a better (simpler) name

Comment (michaelneale) [reply Jenkinsfile]: @AlanFoster this is just for debugging anyway...

**Comment by michaelneale** [comment]
closing as this was just used to diagnose things anyway

**Pull request closed without merging**